src/main/java/com/Acrobot/ChestShop/Listeners/Modules/PriceRestrictionModule.java

```diff
@@ -89,7 +112,9 @@ private void convertToMaterial(String sectionPath) {
             for (String typeId : section.getKeys(false)) {
                 Material material = Material.matchMaterial(typeId);
                 if (material != null) {
-                    configuration.set(sectionPath + "." + material.toString().toLowerCase(Locale.ROOT), configuration.get(sectionPath + "." + typeId));
+                    configuration.set(
+                            sectionPath + "." + material.toString().toLowerCase(Locale.ROOT),
+                            configuration.get(sectionPath + "." + typeId));
```

Comment:
Same line changes here that don't seem to change functionality.

Reply:
Changed - will let you mark as resolved on evaluation